From 2ea23ef6adc9e4d6114345b4682af95b6d8be8bf Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 10 Aug 2012 15:59:05 +0800
Subject: [PATCH 2/4] ENGR00217633 Add new usage to indicate allocate
 contiguous memory

Add new usage to indicate allocate contiguous memory

Change-Id: I5f2b42d192f411300678dff7f1495856a71a98f0
Signed-off-by: Richard Liu <r66033@freescale.com>
---
 include/hardware/gralloc.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/hardware/gralloc.h b/include/hardware/gralloc.h
index b2562e6..faededb 100644
--- a/include/hardware/gralloc.h
+++ b/include/hardware/gralloc.h
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+/*Copyright 2009-2012 Freescale Semiconductor, Inc.*/
 
 #ifndef ANDROID_GRALLOC_INTERFACE_H
 #define ANDROID_GRALLOC_INTERFACE_H
@@ -113,6 +114,9 @@ enum {
      */
     GRALLOC_USAGE_EXTERNAL_DISP         = 0x00002000,
 
+    /* Indicate the buffer will be physical continue, some hw such as vpu need this kind of memory as it does not has mmu */
+    GRALLOC_USAGE_FORCE_CONTIGUOUS = 0x08000000,
+
     /* Must have a hardware-protected path to external display sink for
      * this buffer.  If a hardware-protected path is not available, then
      * either don't composite only this buffer (preferred) to the
-- 
1.8.0

